deepwell/src/models/user.rs

```diff
@@ -24,6 +24,7 @@ pub struct Model {
     pub last_renamed_at: Option<OffsetDateTime>,
     #[sea_orm(column_type = "Text")]
     pub email: String,
+    pub email_is_alias: Option<bool>,
```

Comment:
So the three options here are:
* `true` -- Email is known to be an alias. (e.g. `foo+name@gmail.com`)
* `false` -- Email is not known to be an alias.
* `null` -- Email has not been run through the verification service.
?

Reply:
Yep.

Reply:
Oh I forgot to mention in my PR, you need to add this column to the [migration](https://github.com/scpwiki/wikijump/blob/develop/deepwell/migrations/20220906103252_deepwell.sql#L17) too.